### ISSUE TITLE
Drop Coverity badge

### ISSUE THREAD

**Issue by GregoryLundberg** [wesnoth/wesnoth#4114]
No sense claiming a failure on a service which we no longer use, maintain or are interested in. It's been a "Failure" for a very, very long time.

**Comment by jostephd** [comment]
Or we could start using it again? But I guess there's some history here that I'm unaware of.

**Comment by GregoryLundberg** [comment]
Nah. I think it borked when we rolled back last year to bring master back to 1.14+dev and re-start 1.15. Someone with owner-rights would need to un-bork it. All that occurred during a major outage at Coverity (which lasted weeks: they tried to upgrade something and crashed it all). So when we were ready, they weren't and when they finally were, I guess we'd loss interest.

It's not that it tells us much. AI and I got it down to about a dozen false-positives or upstream issues (mainly with the C++ runtime libraries), plus another dozen or so un-caught exception comments (mainly due to the break in C++ control-flow due to passing through Lua). Maybe once or twice a year an actual issue would crop up from the static analysis.
